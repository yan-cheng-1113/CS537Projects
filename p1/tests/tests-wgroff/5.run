cp /p/course/cs537-oliphant/tests/P1/tests-wgroff/underline.txt .; ./wgroff underline.txt; cat underlineExample.1
