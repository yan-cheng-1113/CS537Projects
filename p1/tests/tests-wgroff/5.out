underlineExample(1)                                          underlineExample(1)

[1mNAME[0m
       underlineExample - an example program

[1mSYNOPSIS[0m
       underlineExample -k -W [options] argument

[1mDESCRIPTION[0m
       This is a fake example program.  The command line switches, options, and
       arguments don't actually exist since this is just an example.
       
       This text is in a new paragraph.  Blank lines should also be output.
       So this [4mparagraph[0m should have a blank line before it.
       You should see bold, italic, and [4munderline[0m and
       these can be [4mcombined[0m.  You can also output __ characters.
       
       Typical man pages will have a NAME section, a SYNOPSIS section,
       and a DESCRIPTION section (at least).  The NAME and DESCRIPTION
       sections are important for the apropos program.
       
       blah.
                                   2023-08-11                                   
